apisix/core/request.lua

```diff
@@ -138,6 +145,15 @@ function _M.set_header(ctx, header_name, header_value)
     end
 end
 
+function _M.add_header(header_name, header_value)
+    local err
+    header_name, err = _validate_header_name(header_name)
+    if err then
+        error(err)
+    end
+
+    req_add_header(header_name,header_value)
```

Comment:
Please ensure a space is after the ','. Let's deal with similar places in this PR.

Reply:
done